common/controllers/framework/framework-step.test.js

```diff
@@ -1,11 +1,14 @@
 const FormWizardController = require('../../../common/controllers/form-wizard')
 const fieldHelpers = require('../../../common/helpers/field')
 const frameworksHelpers = require('../../../common/helpers/frameworks')
+
+const youthRiskAssessmentService = {
+  respond: sinon.stub(),
+}
 const services = {
-  'person-escort-record': require('../../services/person-escort-record'),
-  'youth-risk-assessment': require('../../services/youth-risk-assessment'),
+  personEscortRecord: require('../../services/person-escort-record'),
+  youthRiskAssessment: youthRiskAssessmentService,
```

Comment:
These probably want to match the controller now.

Reply:
I think this is correct, this `services` mocks `req.services`

Reply:
I wonder if this whole thing can be simplified and not need to fake importing them?

And create the stub on `req.services` directly?

Reply:
Yeah, I'll look at different approaches in my next PR, there's a couple of places we do this.